src/components/MessageList/MessageList.tsx

```diff
@@ -249,6 +251,7 @@ export const MessageList = <
     onThreadSelect,
     setFlatListRef,
     threadList,
+    invertedList = true,
```

Comment:
could you put this in alphabetical order with the rest of the props
```suggestion
    inverted = true,
```